Error Installing C++ for Ethereum on Mac

Mac OS High Sierra Version: 10.13.3 (17D102)

```
Chases-MacBook-Pro-2017:~ chase$ brew tap ethereum/ethereum
Chases-MacBook-Pro-2017:~ chase$ brew install cpp-ethereum
brew linkapps cpp-ethereum
==> Installing cpp-ethereum from ethereum/ethereum
==> Cloning https://github.com/ethereum/cpp-ethereum.git
Cloning into '/Users/chase/Library/Caches/Homebrew/cpp-ethereum--git'...
remote: Counting objects: 614, done.
remote: Compressing objects: 100% (590/590), done.
remote: Total 614 (delta 80), reused 156 (delta 9), pack-reused 0
Receiving objects: 100% (614/614), 1.20 MiB | 4.57 MiB/s, done.
Resolving deltas: 100% (80/80), done.
==> Checking out branch develop
==> cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG 
Last 15 lines from /Users/chase/Library/Logs/Homebrew/cpp-ethereum/01.cmake:
    https://github.com/ruslo/hunter/wiki/error.internal

CMake Error at .brew_home/.hunter/_Base/Download/Hunter/0.20.9/8f7cf6d/Unpacked/cmake/modules/hunter_wiki.cmake:12 (message):
Call Stack (most recent call first):
  .brew_home/.hunter/_Base/Download/Hunter/0.20.9/8f7cf6d/Unpacked/cmake/modules/hunter_internal_error.cmake:13 (hunter_wiki)
  .brew_home/.hunter/_Base/Download/Hunter/0.20.9/8f7cf6d/Unpacked/cmake/modules/hunter_calculate_toolchain_sha1.cmake:108 (hunter_internal_error)
  .brew_home/.hunter/_Base/Download/Hunter/0.20.9/8f7cf6d/Unpacked/cmake/modules/hunter_apply_gate_settings.cmake:116 (hunter_calculate_toolchain_sha1)
  .brew_home/.hunter/_Base/Download/Hunter/0.20.9/8f7cf6d/Unpacked/cmake/modules/hunter_finalize.cmake:73 (hunter_apply_gate_settings)
  .brew_home/.hunter/_Base/Download/Hunter/0.20.9/8f7cf6d/Unpacked/cmake/modules/hunter_add_package.cmake:24 (hunter_finalize)
  CMakeLists.txt:90 (hunter_add_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/cpp-ethereum-20180330-41832-enzfhg/CMakeFiles/CMakeOutput.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/ethereum/homebrew-ethereum/issues

Chases-MacBook-Pro-2017:~ chase$ brew linkapps cpp-ethereum
Warning: Calling 'brew linkapps' is deprecated!
There is no replacement.
/usr/local/Homebrew/Library/Homebrew/cmd/linkapps.rb:18:in `linkapps'

Error: No such keg: /usr/local/Cellar/cpp-ethereum
```

Is this the same issue?  Work-arounds welcome.

```sh
$ brew install cpp-ethereum
==> Installing cpp-ethereum from ethereum/ethereum
==> Installing dependencies for ethereum/ethereum/cpp-ethereum: cmake, gperftools, snappy, leveldb
==> Installing ethereum/ethereum/cpp-ethereum dependency: cmake
==> Downloading https://homebrew.bintray.com/bottles/cmake-3.11.2.high_sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring cmake-3.11.2.high_sierra.bottle.tar.gz
==> Caveats
Emacs Lisp files have been installed to:
  /usr/local/share/emacs/site-lisp/cmake
==> Summary
/usr/local/Cellar/cmake/3.11.2: 2,363 files, 32.7MB
==> Installing ethereum/ethereum/cpp-ethereum dependency: gperftools
==> Downloading https://homebrew.bintray.com/bottles/gperftools-2.7.high_sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring gperftools-2.7.high_sierra.bottle.tar.gz
/usr/local/Cellar/gperftools/2.7: 103 files, 4.3MB
==> Installing ethereum/ethereum/cpp-ethereum dependency: snappy
==> Downloading https://homebrew.bintray.com/bottles/snappy-1.1.7_1.high_sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring snappy-1.1.7_1.high_sierra.bottle.tar.gz
/usr/local/Cellar/snappy/1.1.7_1: 18 files, 118KB
==> Installing ethereum/ethereum/cpp-ethereum dependency: leveldb
==> Downloading https://homebrew.bintray.com/bottles/leveldb-1.20_2.high_sierra.bottle.tar.gz
######################################################################## 100.0%
==> Pouring leveldb-1.20_2.high_sierra.bottle.tar.gz
/usr/local/Cellar/leveldb/1.20_2: 26 files, 1.2MB
==> Installing ethereum/ethereum/cpp-ethereum
==> Cloning https://github.com/ethereum/cpp-ethereum.git
Cloning into '/Users/jeff/Library/Caches/Homebrew/cpp-ethereum--git'...
remote: Counting objects: 613, done.
remote: Compressing objects: 100% (576/576), done.
remote: Total 613 (delta 84), reused 167 (delta 22), pack-reused 0
Receiving objects: 100% (613/613), 1.19 MiB | 8.32 MiB/s, done.
Resolving deltas: 100% (84/84), done.
==> Checking out branch develop
==> cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/cpp
Last 15 lines from /Users/jeff/Library/Logs/Homebrew/cpp-ethereum/01.cmake:
    https://github.com/ruslo/hunter/wiki/error.internal
-------------------------------------------------------------------

CMake Error at .brew_home/.hunter/_Base/Download/Hunter/0.20.73/5cd594b/Unpacked/cmake/modules/hunter_wiki.cmake:12 (message):
Call Stack (most recent call first):
  .brew_home/.hunter/_Base/Download/Hunter/0.20.73/5cd594b/Unpacked/cmake/modules/hunter_internal_error.cmake:13 (hunter_wiki)
  .brew_home/.hunter/_Base/Download/Hunter/0.20.73/5cd594b/Unpacked/cmake/modules/hunter_calculate_toolchain_sha1.cmake:108 (hunter_internal_error)
  .brew_home/.hunter/_Base/Download/Hunter/0.20.73/5cd594b/Unpacked/cmake/modules/hunter_apply_gate_settings.cmake:116 (hunter_calculate_toolchain_sha1)
  .brew_home/.hunter/_Base/Download/Hunter/0.20.73/5cd594b/Unpacked/cmake/modules/hunter_finalize.cmake:73 (hunter_apply_gate_settings)
  .brew_home/.hunter/_Base/Download/Hunter/0.20.73/5cd594b/Unpacked/cmake/modules/hunter_add_package.cmake:24 (hunter_finalize)
  CMakeLists.txt:87 (hunter_add_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/cpp-ethereum-20180524-72063-1yj2pi/CMakeFiles/CMakeOutput.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/ethereum/homebrew-ethereum/issues

```

```

MacBook-Pro-de-Alailson:ethereum alailsonribeiro$ brew install --env=std cpp-ethereum
Updating Homebrew...
==> Installing cpp-ethereum from ethereum/ethereum
==> Cloning https://github.com/ethereum/aleth.git
Updating /Users/alailsonribeiro/Library/Caches/Homebrew/cpp-ethereum--git
==> Checking out tag v1.4.0rc5
HEAD is now at 9bc368d Aleth 1.4.0rc5
HEAD is now at 9bc368d Aleth 1.4.0rc5
Entering 'evmc'
Entering 'evmjit'
Synchronizing submodule url for 'evmc'
Entering 'evmjit/evmc'
Entering 'hera'
Synchronizing submodule url for 'cmake/cable'
Synchronizing submodule url for 'evm2wasm'
Synchronizing submodule url for 'evmc'
Entering 'hera/cmake/cable'
Entering 'hera/evm2wasm'
Entering 'hera/evmc'
Entering 'test/jsontests'
/Users/alailsonribeiro/Library/Caches/Homebrew/cpp-ethereum--git/evmc
/Users/alailsonribeiro/Library/Caches/Homebrew/cpp-ethereum--git/evmjit
/Users/alailsonribeiro/Library/Caches/Homebrew/cpp-ethereum--git/evmjit/evmc
/Users/alailsonribeiro/Library/Caches/Homebrew/cpp-ethereum--git/hera
/Users/alailsonribeiro/Library/Caches/Homebrew/cpp-ethereum--git/hera/cmake/cable
/Users/alailsonribeiro/Library/Caches/Homebrew/cpp-ethereum--git/hera/evm2wasm
/Users/alailsonribeiro/Library/Caches/Homebrew/cpp-ethereum--git/hera/evmc
/Users/alailsonribeiro/Library/Caches/Homebrew/cpp-ethereum--git/test/jsontests
==> cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/cpp-ethereum/1.4.0rc5 -DCMAKE_BUILD_TYPE=Release -DCMAKE_FIND_FRAMEWO
Last 15 lines from /Users/alailsonribeiro/Library/Logs/Homebrew/cpp-ethereum/01.cmake:
    https://github.com/ruslo/hunter/wiki/error.internal
-------------------------------------------------------------------

CMake Error at .brew_home/.hunter/_Base/Download/Hunter/0.21.16/aa1facc/Unpacked/cmake/modules/hunter_wiki.cmake:12 (message):
Call Stack (most recent call first):
  .brew_home/.hunter/_Base/Download/Hunter/0.21.16/aa1facc/Unpacked/cmake/modules/hunter_internal_error.cmake:13 (hunter_wiki)
  .brew_home/.hunter/_Base/Download/Hunter/0.21.16/aa1facc/Unpacked/cmake/modules/hunter_calculate_toolchain_sha1.cmake:108 (hunter_internal_error)
  .brew_home/.hunter/_Base/Download/Hunter/0.21.16/aa1facc/Unpacked/cmake/modules/hunter_apply_gate_settings.cmake:116 (hunter_calculate_toolchain_sha1)
  .brew_home/.hunter/_Base/Download/Hunter/0.21.16/aa1facc/Unpacked/cmake/modules/hunter_finalize.cmake:73 (hunter_apply_gate_settings)
  .brew_home/.hunter/_Base/Download/Hunter/0.21.16/aa1facc/Unpacked/cmake/modules/hunter_add_package.cmake:24 (hunter_finalize)
  CMakeLists.txt:71 (hunter_add_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/cpp-ethereum-20180830-18435-42jnpa/CMakeFiles/CMakeOutput.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/ethereum/homebrew-ethereum/issues

```

Facing the same error with Aleth version 1.5.2, any updates on this.

'==> Checking out tag v1.5.2
HEAD is now at 2a2e584 Aleth 1.5.2
Submodule 'cable' (https://github.com/ethereum/cable) registered for path 'cmake/cable'
Submodule 'evmc' (https://github.com/ethereum/evmc) registered for path 'evmc'
Submodule 'test/jsontests' (https://github.com/ethereum/tests.git) registered for path 'test/jsontests'
Cloning into '/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/cmake/cable'...
Cloning into '/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/evmc'...
Cloning into '/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/test/jsontests'...
Submodule path 'cmake/cable': checked out 'cd585c5a70374f335336cd3240fcd22f64d794ed'
Submodule path 'evmc': checked out '354ba6f540655649e081fb455e21998186bf5576'
Submodule path 'test/jsontests': checked out '4cfaa3e600f45d34bc00d88a4b142d2f27762912'
/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/cmake/cable
/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/evmc
/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/test/jsontests
==> cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG 
Last 15 lines from /Users/kaustubh/Library/Logs/Homebrew/cpp-ethereum/01.cmake:
    https://github.com/ruslo/hunter/wiki/error.internal
-------------------------------------------------------------------

CMake Error at .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_wiki.cmake:12 (message):
Call Stack (most recent call first):
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_internal_error.cmake:13 (hunter_wiki)
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_calculate_toolchain_sha1.cmake:110 (hunter_internal_error)
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_apply_gate_settings.cmake:135 (hunter_calculate_toolchain_sha1)
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_finalize.cmake:73 (hunter_apply_gate_settings)
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_add_package.cmake:23 (hunter_finalize)
  CMakeLists.txt:65 (hunter_add_package)

-- Configuring incomplete, errors occurred!
See also "/tmp/cpp-ethereum-20190307-31013-1km0jj0/CMakeFiles/CMakeOutput.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/ethereum/homebrew-ethereum/issues'

Facing the same problem while installing Aleth 1.5.2, Any updates on how to resolve this issue.

> ==> Checking out tag v1.5.2
HEAD is now at 2a2e584 Aleth 1.5.2
Submodule 'cable' (https://github.com/ethereum/cable) registered for path 'cmake/cable'
Submodule 'evmc' (https://github.com/ethereum/evmc) registered for path 'evmc'
Submodule 'test/jsontests' (https://github.com/ethereum/tests.git) registered for path 'test/jsontests'
Cloning into '/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/cmake/cable'...
Cloning into '/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/evmc'...
Cloning into '/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/test/jsontests'...
Submodule path 'cmake/cable': checked out 'cd585c5a70374f335336cd3240fcd22f64d794ed'
Submodule path 'evmc': checked out '354ba6f540655649e081fb455e21998186bf5576'
Submodule path 'test/jsontests': checked out '4cfaa3e600f45d34bc00d88a4b142d2f27762912'
/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/cmake/cable
/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/evmc
/Users/kaustubh/Library/Caches/Homebrew/cpp-ethereum--git/test/jsontests
==> cmake . -DCMAKE_C_FLAGS_RELEASE=-DNDEBUG -DCMAKE_CXX_FLAGS_RELEASE=-DNDEBUG 
Last 15 lines from /Users/kaustubh/Library/Logs/Homebrew/cpp-ethereum/01.cmake:
    https://github.com/ruslo/hunter/wiki/error.internal
-------------------------------------------------------------------

CMake Error at .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_wiki.cmake:12 (message):
Call Stack (most recent call first):
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_internal_error.cmake:13 (hunter_wiki)
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_calculate_toolchain_sha1.cmake:110 (hunter_internal_error)
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_apply_gate_settings.cmake:135 (hunter_calculate_toolchain_sha1)
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_finalize.cmake:73 (hunter_apply_gate_settings)
  .brew_home/.hunter/_Base/Download/Hunter/0.23.44/c4cfcc0/Unpacked/cmake/modules/hunter_add_package.cmake:23 (hunter_finalize)
  CMakeLists.txt:65 (hunter_add_package)


-- Configuring incomplete, errors occurred!
See also "/tmp/cpp-ethereum-20190307-31013-1km0jj0/CMakeFiles/CMakeOutput.log".

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/ethereum/homebrew-ethereum/issues

--- a/README.md
+++ b/README.md
@@ -3,7 +3,7 @@ homebrew-ethereum
 
 Homebrew Tap for Ethereum
 
-**Important note: reporting issues with any of these brews should be done at their respective repositories ([Go client](https://github.com/ethereum/go-ethereum), [C++ client](https://github.com/ethereum/cpp-ethereum) and [Solidity](https://github.com/ethereum/solidity)).**
+**Important note: reporting issues with any of these brews should be done at their respective repositories ([Go client](https://github.com/ethereum/go-ethereum) and [Solidity](https://github.com/ethereum/solidity)).**
 
 ## Installation
 
@@ -16,11 +16,6 @@ brew tap ethereum/ethereum
 brew install ethereum
 ```
 
-### C++ client
-```
-brew install cpp-ethereum
-```
-
 ### Solidity
 ```
 brew install solidity
@@ -31,37 +26,11 @@ brew install solidity
 ### Go client
 `geth`
 
-### C++ client
-`eth`
-
 ### Solidity
 `solc`
 
 ## Development
-Get the latest development version with the `--devel` flag. Use `--build-from-source` if you don't want a pre-built bottle. Alternatively you can use the `--successful` flag (cpp-ethereum only, see [important note below](#important-note-when-using---successful)) or any other [available options](#options).
-
-### C++ client
-```
-brew reinstall cpp-ethereum --devel
-```
-
-For the latest successful build on develop (last successful build from [cpt-obvious](https://build.ethdev.com/waterfall)):
-```
-brew reinstall cpp-ethereum --devel --successful
-```
-
-#### Important note when using --successful
-
-If you get an error looking like this:
-```
-==> Cloning https://github.com/ethereum/cpp-ethereum.git
-Updating /Library/Caches/Homebrew/ethereum--git
-fatal: reference is not a tree: <latest commit hash>
-Error: Failed to download resource "ethereum"
-Failure while executing: git checkout -q -f
-```
-
-Either try `brew fetch cpp-ethereum --devel` or simply delete the cache with `rm -rf /Library/Caches/Homebrew/cpp-ethereum--git`
+Get the latest development version with the `--devel` flag.
 
 
 ### Go client
@@ -74,10 +43,6 @@ brew reinstall ethereum --devel
 
 Go:
 * `--devel` is on develop branch
-* normal install is on master branch
-
-C++:
-* `--devel` is on [develop](https://github.com/ethereum/cpp-ethereum/commits/develop)
 * normal install is on master branch
 
 
@@ -94,17 +59,11 @@ brew update && brew upgrade
 brew update && brew reinstall ethereum
 ```
 
-### C++ client
-```
-brew update && brew reinstall cpp-ethereum
-```
-
 
 ## Versions
 List available versions with:
 ```
 ls -l /usr/local/Cellar/ethereum
-ls -l /usr/local/Cellar/cpp-ethereum
 ```
 
 If you have other versions installed, you can switch with:

--- a/cpp-ethereum.rb
+++ b/cpp-ethereum.rb
@@ -17,24 +17,7 @@ class CppEthereum < Formula
   desc "Ethereum C++ client"
   homepage "http://cpp-ethereum.org"
 
-  url "https://github.com/ethereum/aleth.git", :tag => "v1.5.2"
-  version "1.5.2"
+  url "https://github.com/ethereum/aleth.git"
 
-  devel do
-    url "https://github.com/ethereum/aleth.git", :branch => "master"
-    version "development"
-  end
-
-  depends_on "cmake" => :build
-  depends_on "leveldb"
-
-  def install
-    system "cmake", ".", *std_cmake_args, "-DVMTRACE=ON"
-    system "make"
-    system "make", "install"
-  end
-
-  test do
-    system "#{bin}/aleth", "--version"
-  end
+  odie "Aleth (formerly cpp-ethereum) has been removed from Homebrew. Please install binary releases from https://github.com/ethereum/aleth/releases."
 end

